### ISSUE TITLE
Change table text to white when background colour is dark

### ISSUE THREAD

**Issue by bhavyaac** [fungenomics/braindex#32]
Some cluster names are very hard to read because of dark-coloured backgrounds:
![image](https://user-images.githubusercontent.com/26724860/122983422-fcb9e480-d369-11eb-9ceb-1d8c91c3785e.png)

Since the table is formatted using formatStyle(), I can try to use styleInterval() to change the text colour to white for some backgrounds. I would need to make a list of clusters that are associated with dark palette colours (using dark() in functions.R which I used for tooltips), and pass it as input to the "cuts" for styleInterval(). See examples of styleInterval() usage here: https://rstudio.github.io/DT/010-style.html

**Comment by sjessa** [comment]
@bhavyaac Alternative idea -- what about leaving the "Cluster" column un-coloured, and having a second column "Color", which has no text, but just a colored background, so no text changes are needed? (Less elegant, but maybe simpler!)

**Comment by bhavyaac** [comment]
Solved this by using reactable to display the table instead of DT. These tables have more powerful [options for conditional styling](https://glin.github.io/reactable/articles/conditional-styling.html) as well as the ability to link the table to a marker table using Crosstalk. I also found it much more intuitive than DT for formatting.
![image](https://user-images.githubusercontent.com/26724860/124677429-7388ce80-de8e-11eb-9dd4-54a5d2f95d4f.png)
